src/status_im2/contexts/quo_preview/links/internal_link_card_community.cljs

```diff
@@ -0,0 +1,37 @@
+(ns status-im2.contexts.quo-preview.links.internal-link-card-community
+  (:require
+    [quo.core :as quo]
+    [reagent.core :as reagent]
+    [status-im2.common.resources :as resources]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:type :text :key :title}
+   {:type :text :key :description}
+   {:type :boolean :key :loading?}
+   {:type :text :key :members-count}
+   {:type :text :key :active-members-count}])
+
+(defn view
+  []
+  (let [state (reagent/atom
+               {:title                "Doodles"
+                :description          "Coloring the world with joy • ᴗ •"
+                :members-count        24
+                :active-members-count 12
+                :link                 "https://status.app/community-link"
+                :loading?             false})]
+    (fn []
+      [preview/preview-container
+       {:state      state
+        :descriptor descriptor}
+       [quo/internal-link-card
+        {:title                (:title @state)
+         :description          (:description @state)
+         :icon                 (resources/get-mock-image :status-logo)
+         :banner               (resources/get-mock-image :dark-blur-background)
+         :loading?             (:loading? @state)
+         :members-count        (int (:members-count @state))
+         :loading-message      (:loading-message @state)
+         :active-members-count (int (:active-members-count @state))
```

Comment:
See if you can use `:type :number` instead of `:type :text` in the number fields, it should work. Then you don't need to manually call `int`.

Once you have that, you would probably be able to simplify the code because there's no need to manually get every deref'ed value to build a new map:

```clojure
[quo/internal-link-card
 (assoc @state
        :icon     (resources/get-mock-image :status-logo)
        :banner   (resources/get-mock-image :dark-blur-background)
        :on-clear #(js/alert "Clear button pressed"))]
```